.github/workflows/pr-gpu.yaml

```diff
@@ -4,7 +4,7 @@ on:
     branches:
     - main
     - release/*
-  pull_request_target:
+  pull_request:
```

Comment:
Why was this changed?

Reply:
This is a temporary change to let the tests run -- messing with the GPU tests is finicky. @vchiley  will change it back before it merges

Reply:
This was to enable both torch2.0.1 and torch1.13.1 tests. 
This will be undone before merge.